README.md

```diff
@@ -1,70 +1,49 @@
-IBM_DB_SA
+[![Build Status](https://travis-ci.com/IBM/sqlalchemy-ibmi.svg?branch=master)](https://travis-ci.com/IBM/sqlalchemy-ibmi)
+[![Latest version released on PyPi](https://img.shields.io/pypi/v/sqlalchemy-ibmi.svg)]()
+[![](https://img.shields.io/pypi/pyversions/sqlalchemy-ibmi.svg)]()
+[![Documentation Status](https://readthedocs.org/projects/sqlalchemy-ibmi/badge/?version=latest)]()
```

Comment:
Since these don't exist, let's omit them. We can create issues to get them set up.